bin/varnishncsa/varnishncsa.c

```diff
@@ -160,8 +160,26 @@ static struct ctx {
 	const char		*handling;
 	const char		*side;
 	int64_t			vxid;
+	int			vcl_recv_seen;
+	int			vcl_br_seen;
```

Comment:
Wouldn't it be better to add some new vsl tag that indicates the end of reception from a peer so that we can rely on it in varnishncsa and avoid taking headers added by the core code before vcl_recv/vcl_backend_response into account ?

Reply:
We should focus on the new model for NCSA formats here.

Reply:
updated to `recd_req` and `recd_beresp`